quickstart/docker/all-in-one/README.md

```diff
@@ -16,13 +19,18 @@ This is the primary use case for this project: running the `ziti edge quickstart
 2. Run the project.
 
     ```bash
-    docker compose up --detach
+    docker compose up
     ```
 
-3. Modify the state in `./persistent/`, and bounce the container.
+3. Modify configuration and bounce the container.
+
+    If you set `ZITI_HOME=./persistent`, then you would modify the configs in `./persistent/` on the Docker host.
```

Comment:
can we reword this. something like:
> By default the compose file uses will place files into a ./persistent folder adjacent to where you run docker compose from. Set `ZITI_HOME` to change the location where the image stores the persistet files.

something like that?

Reply:
That would be accurate if inverted like this:

- default: named volume managed by Compose, 
- alternative: bind volume from Docker host's filesystem

> By default, the Compose project places files into `/persistent` folder in a volume that's mounted on the container and managed by Compose. Set `ZITI_HOME` to mount any folder from the Docker host that's writeable by the logged-in user on the container instead of using the Docker-managed volume.